laundry/models.py

```diff
@@ -0,0 +1,25 @@
+import uuid
+
+from django.db import models
+
+
+class LaundrySnapshot(models.Model):
+    hall_id = models.IntegerField()
+    date = models.DateTimeField(auto_now_add=True)
+    available_washers = models.IntegerField()
+    available_dryers = models.IntegerField()
+    total_washers = models.IntegerField()
+    total_dryers = models.IntegerField()
```

Comment:
Since these don't change over time, we could probably just stick them in the `LaundryRoom` model